src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -833,7 +841,8 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
   }
 
   /**
-   * @param newTime - the number of milliseconds to adjust the time by. Can be negative.
```

Comment:
I found this comment misleading - happy to hear thoughts on whether my change is better or not.